tests/resources/test-stations.json

```diff
@@ -403,5 +403,56 @@
     "testStationStatus": "active",
     "testStationTown": "Ruthin",
     "testStationType": "atf"
+  },
+  {
+    "testStationId": "25",
+    "testStationAddress": "1042 Bow Lane",
+    "testStationContactNumber": "02074892500",
+    "testStationEmails": [],
+    "testStationGeneralNotes": "Example of English VEF site with inactive status",
+    "testStationAccessNotes": "",
+    "testStationLatitude": 181,
+    "testStationLongitude": 362,
+    "testStationName": "VEF 1",
+    "testStationPNumber": "V99900",
+    "testStationPostcode": "EC4M 9DT",
+    "testStationCountry": "England",
+    "testStationStatus": "inactive",
+    "testStationTown": "London",
+    "testStationType": "vef"
+  },
+  {
+    "testStationId": "26",
+    "testStationAddress": "3400 Tan-y-Bryn Road",
+    "testStationContactNumber": "01492879456",
+    "testStationEmails": [],
```

Comment:
Might be good to add in some test station emails as a lot of the others have them

Reply:
Added